client/my-sites/stats/activity-log-banner/upgrade-banner.jsx

```diff
@@ -0,0 +1,62 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isJetpackSite } from 'state/sites/selectors';
+import Banner from 'components/banner';
+import {
+	PLAN_PERSONAL,
+	PLAN_JETPACK_PERSONAL,
+	FEATURE_OFFSITE_BACKUP_VAULTPRESS_DAILY,
+	FEATURE_NO_ADS,
+} from 'lib/plans/constants';
+
+class UpgradeBanner extends PureComponent {
```

Comment:
`PureComponent` isn't really needed and I personally tend to avoid it unless I have a specific reason for using it. the reason is that `connect()` by-default makes a component pure the same way and so `PureComponent` not only adds a duplicate prop check but also makes the component definition a little more specific than it needs to be. make your own choice here but that's my 2¢